src/main/resources/Azure_Instance_Details.csv

```diff
@@ -197,4 +197,20 @@ L16s v3,Standard_L16s_v3,16,128.0,1.40,1.40,0.84,5.5
 L32s v3,Standard_L32s_v3,32,256.0,2.79,2.79,1.674,11.00
 L48s v3,Standard_L48s_v3,48,384.0,4.19,4.19,2.514,16.50
 L64s v3,Standard_L64s_v3,64,512.0,5.58,5.58,3.348,22.00
-L80s v3,Standard_L80s_v3,80,640.0,6.98,6.98,4.188,27.50
\ No newline at end of file
+L80s v3,Standard_L80s_v3,80,640.0,6.98,6.98,4.188,27.50
+DC4as v5,Standard_DC4as_v5,4,16.0,0.55,0.55,0.55,1.0
+DC16as v5,Standard_DC16as_v5,16,64.0,2.2,2.2,2.2,4.0
+NC96ads A100 v4,Standard_NC96ads_A100_v4,96,880.0,24.2,24.2,24.2,44.0
+NC48ads A100 v4,Standard_NC48ads_A100_v4,48,440.0,11,11,11,20.0
+DC8as v5,Standard_DC8as_v5,8,32.0,1.1,1.1,1.1,2.0
+NV36ads A10 v5,Standard_NV36ads_A10_v5,36,440.0,2.2,2.2,2.2,4.0
+NV36adms A10 v5,Standard_NV36adms_A10_v5,36,880.0,3.3,3.3,3.3,6.0
+NV72ads A10 v5,Standard_NV72ads_A10_v5,72,880.04.4,4.4,4.4,8.0

```

Comment:
@mohanbaabu1996 I think there's something wonky with this line

Reply:
@gueniai Updated Standard_NV72ads_A10_v5 types, missed a comma